### ISSUE TITLE
Add `gt` and `gtsummary` examples to regression output vignette

### ISSUE THREAD

**Issue by lukesonnet** [DeclareDesign/estimatr#287]
This will be done once those packages are on CRAN (tagging @vincentarelbundock)

**Comment by lukesonnet** [comment]
Is the gtsummary on cran the same package? I can't see author overlap... @vincentarelbundock

**Comment by vincentarelbundock** [comment]
Indeed, there is an unfortunate name conflict. I released my package on github in Jan 2019, but had held off on releasing to CRAN because {gt} isn't there yet, and CRAN doesn't like off-site dependencies.

As far as I can tell @ddsjoberg developed a package called {clintable} based on {kableExtra} tables. A couple weeks ago, he appears to have changed the name of his package to {gtsummary}. As far as I can tell (I may be wrong), that package doesn't actually use {gt} yet, so perhaps they released in in anticipation of using {gt} in the future (name squat?).

The name conflict has already led someone to raise an issue on my preferred; which...argh! Given that I released my package as "gtsummary" publicly before, my preferred resolution would be for @ddsjoberg to change the name back to the original (or to pick another one), but I obviously don't have any control over that. I sent him an email 4 days ago and am waiting for a reply.

**Comment by acoppock** [comment]
Ouf, that's a really unfortunate situation and I'm really sorry to hear about this conflict. @lukesonnet let's hold off until this is resolved. Thank you @vincentarelbundock

**Comment by lukesonnet** [comment]
Any update here? Sorry for the conflict, I just am curious about how best to support this going forward.

**Comment by vincentarelbundock** [comment]
Thanks for following up!

The author does not sound receptive beyond "hoping the conflict does not cause confusion in users" (it already has, obviously). It looks like I'll have to change the name of my package.

I was thinking about `modelsummary` with a shortcut to `ms`.

What do you think? Any other suggestions?

**Comment by vincentarelbundock** [comment]
@lukesonnet @acoppock 

Sorry this took so long, but I just wanted to let you know that the package is now available from CRAN as {modelsummary}.

Thanks again for your interest. Please let me know if you have any feature requests or if you find any bugs.

**Comment by lukesonnet** [comment]
Great! Thanks! We're still gonna wait to push this out until `gt` is on CRAN.
